src/main/java/io/github/moulberry/notenoughupdates/overlays/EquipmentOverlay.java

```diff
@@ -378,6 +383,70 @@ public void renderPets(GuiInventory inventory, int mouseX, int mouseY, int width
 		}
 	}
 
+	private final Map<ItemStack, Integer> itemsToAdd = new HashMap<>();
+
+	@SubscribeEvent
+	public void onClickItem(PlayerInteractEvent event) {
+		if (event.action == PlayerInteractEvent.Action.LEFT_CLICK_BLOCK || Minecraft.getMinecraft().thePlayer.getHeldItem() == null) return;
```

Comment:
Although equivalent, I think in this case it would be better to use != RIGHT_CLICK instead:
```suggestion
		if ((event.action == PlayerInteractEvent.Action.RIGHT_CLICK_BLOCK && event.action == PlayerInteractEvent.Action.RIGHT_CLICK_AIR) || Minecraft.getMinecraft().thePlayer.getHeldItem() == null) return;
```

This conveys that we are specifically looking for a right click. You could also split this up into two ifs, but that is up to you.